src/common/turbo.ts

```diff
@@ -225,3 +279,26 @@ export class TurboArNSPaymentProvider implements ArNSPaymentProvider {
     };
   }
 }
+
+type WanderWallet = {
+  signMessage: (message: Uint8Array) => Promise<Uint8Array>;
+  getActivePublicKey: () => Promise<string>;
+};
+
+function isWanderArweaveBrowserSigner(signer: unknown): signer is WanderWallet {
+  return (
+    typeof signer === 'object' &&
+    signer !== null &&
+    'signMessage' in signer &&
+    'getActivePublicKey' in signer
+  );
+}
+
+export function isTurboArNSSigner(signer: unknown): signer is TurboArNSSigner {
+  const isWanderWallet = isWanderArweaveBrowserSigner(signer);
+  const isSigner =
+    signer instanceof EthereumSigner ||
+    signer instanceof InjectedEthereumSigner ||
+    signer instanceof ArweaveSigner;
```

Comment:
add ArconnectSigner

Reply:
[35ae6aa](https://github.com/ar-io/ar-io-sdk/pull/458/commits/35ae6aa2f2e7a5429c35df9a4a4cabbbb499fed3)

Reply:
does it need to be an instance of one of these, or just have a `sign()` function?